### PR TITLE
Add styled components

### PR DESCRIPTION
**Pull request by BrettEastman**
Added styled components and tested it with a few of my components. See questionsAnswers folder for my style of working with it.

### CODE REVIEW

File: client/source/components/questionsAnswers/InputQuestion.jsx [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+      {/* Product ID from InputQuestion: {prodID} */}
       <input
         type="text"
         placeholder="Have a Question? Search For Answers"
       // onChange={(e) => setQuestInput(e.target.value)}
       // value={questInput}
       />
-      <button type="button" onClick={() => setProdID('new product ID')}>Change ID</button>
-    </div>
+      <button type="button" onClick={() => setProdID('new product ID')}>Go!</button>
```
Comment (AedonGrunGott) [inline client/source/components/questionsAnswers/InputQuestion.jsx]: some commented code broke this line

File: client/source/components/questionsAnswers/DisplayPhotos.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { PhotoContainer } from './styles/Containers.styled';
+
+const DisplayPhotos = function CreateDisplayPhotosComponent() {
+
+  return (
```
Comment (AedonGrunGott) [inline client/source/components/questionsAnswers/DisplayPhotos.jsx:6]: You skipped the linting... didn't you.  ;)

File: client/source/components/questionsAnswers/DisplayPhotos.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <PhotoContainer>
+      <img
+        src="https://images.unsplash.com/photo-1477420143023-6a0e0b04b69a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
```
Comment (AedonGrunGott) [inline client/source/components/questionsAnswers/DisplayPhotos.jsx]: I saw that you had test data. Good idea, so you can avoid hardcoding this here. Pass that test data in as if it were the real deal.

File: client/source/components/questionsAnswers/InputQuestion.jsx [hunk 4]
```diff
@@ -1,21 +1,21 @@
 import React, { useContext } from 'react';
-import ProdContext from '../../ProdContext';
-
+// import ProdContext from '../../ProdContext';
```
Comment (AedonGrunGott) [inline client/source/components/questionsAnswers/InputQuestion.jsx]: Leaking development comments into main! (feel free to bother me about this too!)

File: client/source/components/questionsAnswers/QAModule.jsx [hunk 5]
```diff
@@ -1,22 +1,25 @@
...
+import InputQuestion from './InputQuestion';
+import LoadAddQA from './LoadAddQA';
+import DisplayPhotos from './DisplayPhotos';
+import { Container, InputContainer } from './styles/Containers.styled';
 
 // MAIN
-const QAModule = () => {
-
+const QAModule = function CreateQAModuleComponent() {
   // const [questInput, setQuestInput] = useState('');
```
Comment (AedonGrunGott) [inline client/source/components/questionsAnswers/QAModule.jsx]: leaky comments

File: client/source/components/questionsAnswers/InputQuestion.jsx [hunk 6]
```diff
@@ -1,21 +1,21 @@
...
+import { InputContainer } from './styles/Containers.styled';
 
 const InputQuestion = function CreateInputQuestionComponent() {
-  const { prodID, setProdID } = useContext(ProdContext);
+  // const { prodID, setProdID } = useContext(ProdContext);
```
Comment (AedonGrunGott) [inline client/source/components/questionsAnswers/InputQuestion.jsx]: more leaks!

File: package.json [hunk 7]
```diff
@@ -16,7 +16,8 @@
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
```
Comment (AedonGrunGott) [inline package.json:19]: I get why the trailing-comma rule is a thing now. No issue here, just pointing out that it looks funny in GitHub's PR UI.

**Review by AedonGrunGott — CHANGES_REQUESTED** [review]
Works as expected, and looks like a good way forward.
Though it does look like you somehow evaded our mandatory linter.   ;)

In general it's probably best if we clean up our code before pushing it to main. It could get really cluttered really fast. The best approach while working on an active branch would probably be to make a new branch off of your current active one, clean up any comments and then do a PR off that one. Thus you preserve all your important notes.

File: client/source/components/questionsAnswers/InputQuestion.jsx (hunk 1 above)
Comment (BrettEastman) [reply client/source/components/questionsAnswers/InputQuestion.jsx]: Thank you for all of the great comments. Sorry about the comments, but they have all been taken care of now. Thank you for bringing it to my attention!

**Comment by AedonGrunGott** [comment]
It's late and my 🧠 is fried. I promise to look this over soon!

**Review by AedonGrunGott — APPROVED** [review]
\o/ APPROVED!

**Pull request merged**